i18n/en.json

```diff
@@ -4752,7 +4752,7 @@
   },
   {
     "id": "model.service_terms.is_valid.text.app_error",
-    "translation": "Invalid terms of service text."
+    "translation": "Terms of service text is too long."
```

Comment:
Propose `Custom terms of service text is too long. Maximum {MaxLength} characters allowed`

where `{MaxLength}` is the max character size, I think around 16,000?

Reply:
Done. It's 16384 characters.